### DISCUSSION TITLE
Can I use redux in js plug-in?

### DISCUSSION THREAD

**Discussion by Angry-Sparrow** [reduxjs/redux#4094] [Help] [answered]
Recently, I need to write a plug-in with status.
For some reason, my plug-in can only be implemented in native JS.
Can I use redux in my plugin?
What do I need to pay attention to？
Will it affect the users who use my plug-in？

**Comment by markerikson** [comment, marked as answer]
Redux itself is a plain JS library, so yes, you can.  It won't "affect" anyone, but it is up to you to make sure the code you write and package has all the dependencies needed to run your own code.

**Reply by Angry-Sparrow** [reply]
Thanks！
I‘ll try.
